packages/indy-vdr/src/anoncreds/IndyVdrAnonCredsRegistry.ts

```diff
@@ -390,31 +396,196 @@ export class IndyVdrAnonCredsRegistry implements AnonCredsRegistry {
     }
   }
 
+  public async getRevocationRegistryDefinition(
+    agentContext: AgentContext,
+    revocationRegistryDefinitionId: string
+  ): Promise<GetRevocationRegistryDefinitionReturn> {
+    try {
+      const indySdkPoolService = agentContext.dependencyManager.resolve(IndyVdrPoolService)
+      const did = didFromRevocationRegistryDefinitionId(revocationRegistryDefinitionId)
```

Comment:
You're calling `didFromRevocationRegistryDefinitionId`. I think you can reuse the value

Reply:
@TimoGlastra This is not clear. What do I reuse the value for?

Reply:
I meant to say you're calling `didFromRevocationRegistryDefinitionId` twice